docs/plugins.rst

```diff
@@ -9,16 +9,31 @@ features to Pelican without having to directly modify the Pelican core.
 How to use plugins
 ==================
 
-To load plugins, you have to specify them in your settings file. There are two
-ways to do so. The first method is to specify strings with the path to the
-callables::
+Starting with version 4.3, Pelican moved to a new plugin structure utilizing
+namespace packages. Plugins supporting this structure will install under the
+namespace package ``pelican.plugins`` and can be automatically discovered
+by Pelican.
 
-    PLUGINS = ['package.myplugin',]
+If you leave the ``PLUGINS`` setting as default (``None``), Pelican will then
+collect the namespace plugins and register them. If on the other hand you
+specify a ``PLUGINS`` settings as a list of plugins, this autodiscovery will
+be disabled and only listed plugins will be registered and you will have to
+explicitly list the namespace plugins as well.
 
-Alternatively, another method is to import them and add them to the list::
+If you are using ``PLUGINS`` setting, you can specify plugins in two ways.
+First method is using strings to the plugins. Namespace plugins can be
```

Comment:
First method is using strings to the plugins. → The first method specifies plugins as a list of strings.